start.js

```diff
@@ -0,0 +1,20 @@
+var spawn = require('child_process').spawn
+var os = require('os')
+var start
+
+function onLog (data) {
+  console.log(data.toString())
+}
+
+function onError (data) {
+  console.error(data.toString())
+}
+
+if (os.type() === 'Linux') {
```

Comment:
Sadly I have to inform you, that this isn't working on macOS. require('os').type() === 'Darwin' on macOS.

I would recommend that you check if the os is windows else handle it as unix.
```suggestion
if (os.type() === 'Windows_NT') {
  start = spawn('node', ['bin\\start.js'])
} else {
  start = spawn('./bin/start.js')
}
```

Reply:
sorry I don't know how to delete multiple lines in an review. You have to delete the else if block, too.

Reply:
Fixed